src/util/artifacts.js

```diff
@@ -14,12 +16,36 @@ export async function prepareArtifactsDir(
     const stats = await fs.stat(artifactsDir);
     if (!stats.isDirectory()) {
       throw new UsageError(
-        `--artifacts-dir=${artifactsDir} (this value is not a directory)`);
+        `"${artifactsDir}" exists and it is not a directory.`);
```

Comment:
Any reason why you dropped the prefix? I think the prefix will make it unambiguous that the error is talking about the value passed to `--artifacts-dir`, like:

    `--artifacts-dir="${artifactsDir}" exists and it is not a directory.`

Also, just a nit, but using a two-word conjunction will help to convey that its existence is ok but its file state is not ok:

    `--artifacts-dir="${artifactsDir}" exists but it is not a directory.`